js/tests/unit/collapse.js

```diff
@@ -95,7 +95,41 @@ $(function () {
     $('<div id="test1" class="in"/>')
       .appendTo('#qunit-fixture')
       .on('hidden.bs.collapse', function () {
-        ok($target.hasClass('collapsed'))
+        ok($target.hasClass('collapsed'), 'target has class collapsed')
+        done()
+      })
+
+    $target.click()
+  })
+
+  test('should remove "collapsed" class from all triggers targeting target\'s href when collapse is shown', function (assert) {
```

Comment:
These tests need to be updated to use the new QUnit style. http://qunitjs.com/upgrade-guide-2.x/